ProjectHeads/App.Head.props

```diff
@@ -9,7 +9,7 @@
   </ItemGroup>
 
   <!-- Source generators -->
-  <ItemGroup Condition="'$(TargetFramework)' != '$(WinAppSdkTargetFramework)'">
+  <ItemGroup Condition="'$(IsWinAppSdk)' != 'true'">
```

Comment:
I thought we couldn't use these props for these types of references and have to rely on the TFMs, so shouldn't this switch to the `Contains` like you did above?

Reply:
You're right, project heads don't use the MultiTarget system in quite the same way library projects do, and they don't have access to the `MultiTargetIdentifiers`.

That's why we're defining this and other required properties explicitly in the head, before we import anything.

![image](https://github.com/CommunityToolkit/Tooling-Windows-Submodule/assets/9384894/a8db7b69-eb11-4523-80f6-3b8d2fdc28eb)